KMB/make_KMB_info.py

```diff
@@ -73,22 +76,59 @@ def load_mappings(self, update_mappings):
         """
         socken_file = os.path.join(MAPPINGS_DIR, 'socken.json')
         kommun_file = os.path.join(MAPPINGS_DIR, 'kommun.json')
+        countries_file = os.path.join(MAPPINGS_DIR, 'countries_for_cats.json')
+        taggs_file = os.path.join(MAPPINGS_DIR, 'taggs.json')
```

Comment:
Use `tags_file`, `tags.json` etc. It can be confusing and hard to ctrl+f if you don't remember how those are spelled. Make sure it's consistent throughout the file.